website/templates/project/addon/widget.mako

```diff
@@ -9,14 +9,17 @@
 
                 </div>
             </div>
+    % else:
+        <div>
+    % endif
 
     % if complete:
 
         <div class="panel-body">
             ${self.body()}
         </div>
 
-    % else:
+    % elif not complete and 'write' in user['permissions']:
```

Comment:
Strictly speaking, the `not complete` in this isn't really doing anything, since you wouldn't get to this branch unless complete were false. This could be re-written as `elif 'write' in user['permissions']:`. It's a trivial change either way.